prometheus_query_to_csv/prom_query_to_csv.py

```diff
@@ -0,0 +1,168 @@
+"""
```

Comment:
Nit: we should include a shebang if this is a Python script

`#!/usr/bin/env python3`